nautobot_circuit_maintenance/models.py

```diff
@@ -1,9 +1,11 @@
 """Models for Circuit Maintenance."""
 import hashlib
+import pickle  # nosec
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.urls import reverse
 from django.utils.timezone import now
+from django_cryptography.fields import encrypt
```

Comment:
We're having a lot of ongoing pain with the use of `django_cryptography` in Nautobot core. I'd highly recommend against adding any new uses of this library.

Reply:
which alternative I have to keep a dynamic secret shared across multiple Nautobot instances?

Reply:
IMHO, for the time being just store it in the DB unencrypted. You can make the UI and API refuse to retrieve/render the stored value to the user, if you'd like, as a first layer of defense, but anyone who has direct read access to the DB has enough privileges that I think this would be the least of your concerns security-wise.

In the longer term (i.e. https://github.com/nautobot/nautobot/issues/541) this would be something we'd probably want to reach out to an external system like Vault to store it in.

Reply:
thanks for your feedback!